### PR TITLE
Adds badge description to the README

### LINKED ISSUE #3227: A 'Powered by MDAnalysis' badge
It'd be good to have some sort of feedback on which larger scale projects use MDAnalysis
- as visibility metrics in grant applications;
- for warm fuzzy open-source feelings.

Encouraging our users to display a 'Powered by MDAnalysis' [Shields.io](https://shields.io) badge could serve that purpose, plus giving extra visibility. Here's a starting example, with default gray/orange color combination:

[Powered by MDAnalysis](https://www.mdanalysis.org)

Or, after a bit more work, with the overlaid .ico logo:

EDIT: github seems to have a proxy/caching system for outside image urls and the logo makes this too big. Try [this link instead](https://img.shields.io/badge/Powered%20by-MDAnalysis-orange.svg?logoWidth=15&logo=data:image/x-icon;base64,AAABAAEAISAAAAEAIACoEQAAFgAAACgAAAAhAAAAQAAAAAEAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAkv//AJL//wCS//8Akv//AJL//wCS//8Akv//AJL//wCS//8AAAAAgICA/4CAgP+AgID/gICA/4CAgP+AgID/gICA/4CAgP+AgID/AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAkv9XAJL//QCS//8Akv//AJL//wCS//8Akv//AJL+/wCS/8KWfGlJgICA/ICAgP+AgID/gICA/4CAgP+AgID/gICA/4CAgP6AgICsgICAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAkv5NAJL//QCS//4Akv//AJL//wCS//8Akv//AJH//wCS/7OPfXBhgICA/4CAgP+AgID/gICA/4CAgP+AgID/gICA/4CAgP6AgICmAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAkv8LAJL/9wCR//8Akv/+AJL//wCS//8Akv//AJH+/wKR/EODf3yfgICA/4CAgP+AgID/gICA/4CAgP+AgID/gICA/oCAgP+AgIBmAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACS/wAAAAAAAJL/TACS//kAkv//AJL//wCS//8Akf7/AJL/lZR9ayN/gID7gICA/4CAgP+AgID+gICA/4CAgP+AgID/gICA/4CAgMiAgIABAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACU/yEAkv+qAJH+6QCR/s0Akv9Re4CEJoB/f/OAgID/gYGB/4KCgvyAgID/f39//4CAgP+AgID/gICAr4CAgAkAAAAAgICAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAgIAAAAAAAICAgCKAgIBogICAm4F/foiYfGc7uXdHJKl6VyyFf3p8gICA/X19fcjCwsL/wsLC///////CwsL/XFxceYODg2F/f38tAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgICAW4CAgOqAgID/gICA/4CAgP+AgID/gICA+YCAgP+AgID/fHx8mQMDA8cAAAD6AAAA/wAAAP8AAAD/AAAA5gAAAEMAAAAAAAAAAICAgAAAAAAAAJL/AAAAAAAAAAAAAJL/AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAgIAdgICA7YCAgP6AgID/gICA/4CAgP+AgID/gICA/4CAgP9/f3/pDAwMrgAAAP8AAAD+AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAAJAAAAAAAAAAAAAAAAAJL+DgCR/0EAkv8lAJH+AgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAgIBugICA/4CAgP+AgID/gICA/4CAgP+AgID/gICA/4CAgP/CwsL/AAAA3wAAAP4AAAD/AAAA/wAAAP8AAAD/AAAA/gAAAP8AAACUAAAAAACS/wAAkv8lAJL/4gCS//8Akv//AJH+iAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAgICrgICA/4CAgP+AgID/gICA/4CAgP+AgID/gICA/4CAgP//////AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAADbAAAAAAAAAAAAkv9vAJL+/wCS//4Akv//AJH/6wCS/xQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAgIClgICA/4CAgP+AgID/gICA/4CAgP+AgID/gICA/4CAgP/CwsL/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP4AAADxAAAAAAAAAAAAkv5lAJH//wCS//4Akv//AJL/6wCR/wsAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAgIBXgICA/4CAgP+AgID/gICA/4CAgP+AgID/gICA/4CAgP/CwsL/AAAA0gAAAP4AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP4AAAD2AAAAJwAAAAAAkv4MAJH/0QCS//8Akf7/AJL/SgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAgIARgICA14CAgP+AgID+gICA/oCAgP6AgID/gICA/4CAgP6enp7/HBwcngAAAP8AAAD/AAAA/gAAAP8AAAD+AAAA/wAAAP8AAAD+AAAAhQAAAAAAAAAAAAAAAACS/iQAkf4DAAAAAACS/wAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgICAJYCAgMmAgID/gICA/4CAgP+AgID/gICA+4CAgP+AgID/f39/ywgICHoAAAD5AAAA/wAAAP8AAAD/AAAA/wAAAPUAAAD/AAAA/wAAAGYAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACAgIAAgICAAAAAAACAgIAzgICAcoCAgF+AgIAJAAAAAAAAAACAgIBXf39/54CAgPmenp7/z8/P/8LCwv//////wsLC/56env9dXV1hAwMDsAAAAP8AAAC1AAAAJwAAAAkAAAAMAAAAVAAAAIoAAABwAAAAGgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAgICAAAAAAAAAAAAAgICAD39/f8WAgID/g4OD/4SEhPiBgYH/f39//39/f/+BgYH/h4eHyP////8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/gAAAIgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAkv8AAAAAAAAAAACAgIAAAAAAAICAgAmAgIDxgICA/4CAgP+AgID/gICA/4CAgP9/f3//gICA/4CAgPQAAAD/AAAA/wAAAP8AAAD/AAAA/gAAAP8AAAD+AAAA/wAAAP8AAAB/AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAkv8bAJH+bQCS/z0AAAAACJD2AICAgACAgICTgICA/4CAgP+AgID/gICA/4CAgP+AgID/gICA/n9/f/+zs7P/AAAA8gAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP4AAADZAAAADwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAkv/NAJH//wCS//8Akv4QAAAAAAAAAACAgIBpgICA/oCAgP+AgID/gICA/4CAgP+AgID/gICA/4CAgP7e3t7/AAAA6wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAADzAAAAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAkv/PAJL//wCS//8Akv8PAAAAAAAAAACAgIBSgICA+4CAgP+AgID/gICA/4CAgP+AgID/gICA/4CAgP//////AAAA7AAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAADzAAAAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAkv8cAJL/cACS/0AAAAAAAJL/AAAAAACAgIAmgICA+4CAgP6AgID/gICA/4CAgP+AgID/gICA/n9/f//Pz8//AAAA8gAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP4AAADZAAAADwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAkv4AAAAAAAAAAAAAAAAAAAAAAICAgAAAAAAAgICAqoCAgP+AgID/gICA/oCAgP9/f3//gICA/4CAgPaenp7/AAAA/wAAAP8AAAD/AAAA/gAAAP8AAAD/AAAA/wAAAP8AAAB9AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAN7e3v+CgoL+gICA/39/f/+BgYH/3t7e//////8AAAD/AAAA/wAAAPUAAAD/AAAA/wAAAP8AAAD/AAAA+gAAAIsAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAASAAAAbZ6env/CwsL////////////Pz8//BQUFuQAAAP8AAACrAAAAJQAAABMAAAATAAAASwAAAIIAAABnAAAAHAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAQAAAHIAAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAF8AAAAAAAAAAACR/hQAjvkKAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAewAAAP8AAAD/AAAA/gAAAP8AAAD/AAAA/wAAAP8AAAD+AAAAhAAAAAAAkPwaAJH+uACS/9sAkv/UAJL/bQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAPAAAA3gAAAP4AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP4AAAD2AAAAJgAAAAAAkv/CAJH+/wCS//4Akv/+AJL//QCS/0oAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAaAAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAADwAAAAAACS/xoAkv//AJL//gCS//8Akv//AJL+/wCS/50AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAbAAAA/wAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAP8AAADbAAAAAACS/yUAkv7/AJL//gCS//8Akv//AJL//wCS/6YAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAASAAAA6QAAAP4AAAD/AAAA/wAAAP8AAAD/AAAA/gAAAP8AAACSAAAAAAAAAAAAkv/dAJL//wCS/v4Akv//AJL//wCS/lwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAkgAAAP8AAAD/AAAA/wAAAP8AAAD/AAAA/wAAAPIAAAAKAAAAAAAAAAAAkv8rAJL/1gCS/v8Akv/8AJL+ewCR/wIAkf4AAAAAAAAAAAAAAAAAAAAAAAAAAADgCAP/gAAAAPAIA/+AAAAA8AgD/4AAAADwEAf/gAAAAPgQB/+AAAAA/GAP/4AAAADzwP//gAAAAMAAf/+AAAAAgAA//4AAAACAABw/gAAAAAAAHD+AAAAAAAAcP4AAAACAABx/gAAAAIAAD/+AAAAAwBAP/4AAAAD/wCPfgAAAAP/gAAOAAAAA//AAA4AAAAD/8AABgAAAAPx4AAGAAAAA/HgAAYAAAAD/+AABgAAAAP/4AAOAAAAA//wAA4AAAAD//APfgAAAAP/wD/+AAAAA/+AMf4AAAAD/wBg/gAAAAP/AGB+AAAAA/8AYH4AAAAD/wBg/gAAAAP/APH+AAAAA)
Hopefully, inclusion in a `README.md` would still be possible.

(The base64-encoded .ico is appended to the shields.io url)

Tracking said badges might be as straightforward as crafting a targeted Google query. As an example, for the NumFOCUS badge (#2460): [https://www.google.com/search?q="-NumFOCUS-orange.svg"](https://www.google.com/search?q="-NumFOCUS-orange.svg")

Opinions?

### PR DESCRIPTION
**Pull request by mnmelo**
Fixes #3227

We get a nifty badge!
(Resized from the existing `mdanalysis-logo.ico` to 16x16 and encoded in base64. If left at original size, GitHub doesn't accept the too large resulting url.)

PR Checklist
------------
 - [ ] Tests?
 - [x] Docs?
 - [x] CHANGELOG updated?
 - [x] Issue raised/referenced?

### CODE REVIEW

**Comment by orbeckst** [comment]
Neato: badge

**Review by orbeckst — APPROVED** [review]
nice one, Powered by MDAnalysis

You should write a blog post, too.

**Review by orbeckst — CHANGES_REQUESTED** [review]
Sorry, have witdraw my approval. The failure in https://github.com/MDAnalysis/mdanalysis/pull/3229/checks?check_run_id=2310530477 is actually serious. We cannot use raw html in the README because that will not render on PyPi.

File: README.rst [hunk 1]
```diff
@@ -23,6 +23,25 @@ a `tax-deductible donation`_ to help the project pay for developer time, profess
 
 This project is bound by a `Code of Conduct`_.
 
+If you use MDAnalysis_ in your project consider lettting your users, and the world, know about it by displaying the MDAnalysis_ badge! |powered_by_MDA|
+
+.. raw:: html
```
Comment (orbeckst) [inline README.rst:28]: can't use raw, rewrite with standard reST.

Comment (mnmelo) [reply README.rst:28]: Do we have an all-round compatible way of displaying 'spoiler'-type blocks using standard reST? O I just leave this as a simple code block and hope whatever renders `README.rst` adds a bottom scroller?

Comment (mnmelo) [reply README.rst:28]: At least in GitHub's rendering the code-block approach seems to work. Should I create one block per language (Markdown, reST, html)? Will that be too crowded for README.rst as a landing page? I can have a single block with only the URL, but then there's no way to prod users into actually linking the badge to https://www.mdanalysis.org...

Maybe the best solution is to have an external resource (a gist? the blog post?) listing the implementation in different markups and we just link to it from the README?

Comment (mnmelo) [reply README.rst]: Now linked to the [website](https://www.mdanalysis.org/pages/citations/#powered-by-mdanalysis).

**Comment by mnmelo** [comment]
An RDKit conversion test is failing. Go figure!

**Comment by IAlibay** [comment]
> An RDKit conversion test is failing. Go figure!

Probably #2958 I've restarted CI hopefully it'll pass this time.

**Comment by mnmelo** [comment]
Thanks, @IAlibay. What says you, @orbeckst?

**Review by orbeckst — APPROVED** [review]

**Comment by orbeckst** [comment]
This is the way!

**Comment by orbeckst** [comment]
I tweeted it https://twitter.com/mdanalysis/status/1382006632918392838 but unfortunately the badge screen shot as PNG looks odd on twitter. (SVG didn't work and I don't think you can embed directly in tweets.)

**Pull request merged**